coda/coda_mdstore/views.py

```diff
@@ -589,6 +589,13 @@ def bagURLList(request, identifier, html=False):
         return render(request, 'mdstore/bag_files_download.html',
                       {'links': sorted(transList)})
 
+    if download:
+        zip_filename = 'bag-' + identifier.replace('/', '_') + '.zip'
+        response = StreamingHttpResponse(zip_file_streamer(transList),
+                                         content_type='application/x-zip-compressed')
```

Comment:
Do you know what the purpose of using 'applicaiton/x-zip-compressed' is over 'application/zip'?

Reply:
I'm not sure what the difference is and which is preferred one. I'll look up about it.

Reply:
`application/zip` is a standard Mime type for zip files. `applicaiton/x-zip-compressed` is a variation of it which is used to denote the encoding necessary for the ZIP file format.

Reply:
Is there an assumption that `application/x-zip-compressed` actually makes use of compression in the file (I don't think we are here right?) vs `application/zip`?

Reply:
Honestly I'm not sure if I should be using `application/x-zip-compressed` or `application/zip`.

But, we are using `ZIP_DEFLATED` compression when writing files to `ZipFile`.
`zipstream.ZipFile(mode='w', compression=ZIP_DEFLATED)`

Reply:
I think that we should give `compression= ZIP_STORED` (might be the default value) a try instead of `ZIP_DEFLATED`.  If I understand this correctly, it would store the file as-is in the zip file without additional compression, much like a tar file.  If that seems to work then I think `application/zip` would be just fine for the content_type. 

Another note,  do we need to set `allowZip64=True` (https://docs.python.org/3.8/library/zipfile.html#zipfile-objects) because we will be creating zips bigger than 4Gb sometimes?

Reply:
`allowZip64=True` is the default since Python 3.4.